src/interface/TWNEOAddress.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <TrustWalletCore/TWNEOAddress.h>
+
+#include "../NEO/Address.h"
+
+#include <TrustWalletCore/TWPublicKey.h>
+
+#include <string.h>
+#include <memory>
+
+using namespace TW;
+using namespace TW::NEO;
+
+bool TWNEOAddressEqual(struct TWNEOAddress *_Nonnull lhs, struct TWNEOAddress *_Nonnull rhs) {
+    return lhs->impl == rhs->impl;
+}
+
+bool TWNEOAddressIsValidString(TWString *_Nonnull string) {
+    auto s = reinterpret_cast<const std::string*>(string);
+    return Address::isValid(*s);
+}
+
+struct TWNEOAddress *_Nullable TWNEOAddressCreateWithString(TWString *_Nonnull string) {
+    auto s = reinterpret_cast<const std::string*>(string);
+    const auto address = Address(*s);
```

Comment:
This can throw, let's catch the exception and return `nullptr`

Reply:
done